card/db/schema.rb

```diff
@@ -9,120 +9,120 @@
 
-  create_table "card_acts", :force => true do |t|
-    t.integer  "card_id"
-    t.integer  "actor_id"
+  create_table "card_acts", force: :cascade do |t|
+    t.integer  "card_id",    limit: 4
+    t.integer  "actor_id",   limit: 4
     t.datetime "acted_at"
-    t.string   "ip_address"
+    t.string   "ip_address", limit: 255
   end
 
-  add_index "card_acts", ["actor_id"], :name => "card_acts_actor_id_index"
-  add_index "card_acts", ["card_id"], :name => "card_acts_card_id_index"
+  add_index "card_acts", ["actor_id"], name: "card_acts_actor_id_index", using: :btree
+  add_index "card_acts", ["card_id"], name: "card_acts_card_id_index", using: :btree
 
-  create_table "card_changes", :force => true do |t|
-    t.integer "card_action_id"
-    t.integer "field"
-    t.text    "value"
+  create_table "card_changes", force: :cascade do |t|
+    t.integer "card_action_id", limit: 4
+    t.integer "field",          limit: 4
+    t.text    "value",          limit: 65535
   end
 
-  add_index "card_changes", ["card_action_id"], :name => "card_changes_card_action_id_index"
+  add_index "card_changes", ["card_action_id"], name: "card_changes_card_action_id_index", using: :btree
 
-  create_table "card_references", :force => true do |t|
-    t.integer "referer_id",               :default => 0,  :null => false
-    t.string  "referee_key",              :default => "", :null => false
-    t.integer "referee_id"
-    t.string  "ref_type",    :limit => 1, :default => "", :null => false
-    t.integer "present"
+  create_table "card_references", force: :cascade do |t|
+    t.integer "referer_id",  limit: 4,   default: 0,  null: false
+    t.string  "referee_key", limit: 255, default: "", null: false
+    t.integer "referee_id",  limit: 4
+    t.string  "ref_type",    limit: 1,   default: "", null: false
+    t.integer "present",     limit: 4
   end
 
-  add_index "card_references", ["referee_id"], :name => "card_references_referee_id_index"
-  add_index "card_references", ["referee_key"], :name => "card_references_referee_key_index"
-  add_index "card_references", ["referer_id"], :name => "card_references_referer_id_index"
+  add_index "card_references", ["referee_id"], name: "card_references_referee_id_index", using: :btree
+  add_index "card_references", ["referee_key"], name: "card_references_referee_key_index", using: :btree
+  add_index "card_references", ["referer_id"], name: "card_references_referer_id_index", using: :btree
 
-  create_table "card_revisions", :force => true do |t|
-    t.datetime "created_at", :null => false
-    t.integer  "card_id",    :null => false
-    t.integer  "creator_id", :null => false
-    t.text     "content",    :null => false
+  create_table "card_revisions", force: :cascade do |t|
+    t.datetime "created_at",               null: false
+    t.integer  "card_id",    limit: 4,     null: false
+    t.integer  "creator_id", limit: 4,     null: false
+    t.text     "content",    limit: 65535, null: false
```

Comment:
I'm guessing most of the new limit: values are because of a gem version update, but I do wonder about this one.  Maybe postgres or mysql text is so limited, but I think we want to be careful with any explicit limit on content size.